.travis.yml

```diff
@@ -28,14 +28,10 @@ matrix:
       language: csharp
       solution: Kinvey.sln
       install:
-        - wget -O pubnub.zip https://www.nuget.org/api/v2/package/PubnubNetPlatform/3.8.7
-        - mkdir packages/pubnubnetplatform.3.8.7
-        - mv pubnub.zip packages/pubnubnetplatform.3.8.7/.
-        - unzip packages/pubnubnetplatform.3.8.7/pubnub.zip -d packages/pubnubnetplatform.3.8.7/
         - nuget restore Kinvey.sln
       script:
-        - xbuild Kinvey.Core/Kinvey.Core.csproj
-        - xbuild TestFramework/Tests.Integration/Tests.Integration.csproj
+        - msbuild Kinvey.Core/Kinvey.Core.csproj
+        - msbuild TestFramework/Tests.Integration/Tests.Integration.csproj
```

Comment:
Is the preferred method here to use `msbuild` instead of `dotnet`?

Reply:
we are testing `dotnet` as well, but this one here is to test against mono, which apparently the `msbuild` is the preferred way

Reply:
so basically we are testing in 3 environments for now:
- Linux dotnet
- macOS dotnet
- macOS mono

Ideally we should add more in the future like:
- Windows dotnet
- Xamarin iOS
- Xamarin Android

Reply:
also `xbuild` is now deprecated... i noticed by looking one of the build logs

Reply:
```
>>>> xbuild tool is deprecated and will be removed in future updates, use msbuild instead <<<<
```

Reply:
Ok good. I knew about `xbuild` but didn't know about `mono` preferring `msbuild`.